wordpress-mobile/WordPress-iOS
# Like notifications: show all post and comment likes

Ref: #15662 , #16007
WPKit PR: https://github.com/wordpress-mobile/WordPressKit-iOS/pull/364

This is to merge @bitdc 's feature branch that updates the Likes notifications to show all likes (instead of the previous limit of 10) for post and comment like notifications.

There are  some outstanding tasks, as listed on #15662, but the noticeable ones:
- The users blog is not displayed.
- Tapping the user cell does nothing.
- The Follow button is not displayed. 
  - This will not be added back. The new endpoints don't provide this information, and the new designs have it removed anyway.

To test:

With the `newLikeNotifications` feature flag disabled (the default):
- Go to Notifications > Likes.
- Select a notification.
- Verify the old view is displayed.

With the `newLikeNotifications` feature flag enabled:
- Go to Notifications > Likes.
- Select a notification that has more than 10 likes.
  - Verify the new view is displayed.
  - Verify all the likes are displayed (not limited to 10).
  - Verify the previous/next buttons on nav bar steps through the notifications.
- Select another filter that displays Like notifications (All or Unread).
  - Verify the previous/next buttons on nav bar steps through the notifications.



PR submission checklist:
